src/main/java/Controller/UserAPI.java

```diff
@@ -0,0 +1,122 @@
+package Controller;
+
+import Entity.Class.User;
+import DBAccesser.User.QueryUser;
+import DBAccesser.User.InsertUser;
+import Helper.Helper;
+import Exceptions.RequiredFieldMissingException;
+import Exceptions.UserAlreadyExistsException;
+import Exceptions.InvalidLoginException;
+import Exceptions.UserNotFoundException;
+import Exceptions.APIException;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.http.ResponseEntity;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.ArrayList;
+
+//this annotation tells that this class can contain methods which map to URL requests
+@RestController
+public class UserAPI {
+
+    @Autowired 
+    QueryUser queryUserHelper;
+
+    @Autowired
+    InsertUser insertUserHelper;
+
+    @Autowired
+    Helper helper;
+
+    @ExceptionHandler(UserAlreadyExistsException.class)
+    public ResponseEntity<Object> handleUserAlreadyExistsException(UserAlreadyExistsException e) {
+        return new ResponseEntity<Object>(helper.getResponseBody(e), e.getHttpStatus());
+    }  
+     
+    @ExceptionHandler(RequiredFieldMissingException.class)
```

Comment:
Can change the name of `RequiredFieldMissingException` class to something more specific like `RequiredUserModuleFieldMissingException` so that there's no ambiguity as to how this class is different from other `MissingParameterException` classes in #3